example/Bar13.purs

```diff
@@ -0,0 +1,185 @@
+module Bar13 where
+import Data.Array hiding (init)
+import Data.Maybe
+import Data.Tuple
+import Data.Tuple.Nested
+
+import ECharts.Chart
+import ECharts.Options.Unsafe
+import Data.Maybe
+import Utils
+
+options =
+  {
+    title:
+      {
+        text: "Another chinese stuff",
+        subtext: "Subtitle here"
+      },
+    tooltip:
+      {
+        trigger: "axis",
+        axisPointer:
+          {
+            "type": "cross",
+            lineStyle:
+              {
+                "type": "dashed",
+                width: 1
+              }
+          },
+        formatter: (\p ->
+                     p.seriesName <> " : [ " <>
+                     fromMaybe "" (p.value !! 0) <> ", " <>
+                     fromMaybe "" (p.value !! 1) <> "]")
+      },
+    legend:
+      {
+        "data": ["数据", "数据"]
```

Comment:
I definitely want to use English. But I've found only https://ecomfe.github.io/echarts/doc/example-en.html. I do not speak chinese and my translation would be something like "chinese_word0",
"chinese_word1", "chinese_word2". Should I do it?